packages/webview_flutter/webview_flutter_android/lib/src/android_webview_api_impls.dart

```diff
@@ -239,7 +239,7 @@ class WebSettingsHostApiImpl extends WebSettingsHostApi {
     WebSettings instance,
     bool flag,
   ) {
-    return setJavaScriptCanOpenWindowsAutomatically(
+    return setJavaScriptEnabled(
```

Comment:
This also seems like it should have a test.

Reply:
done